common/cache.go

```diff
@@ -5,33 +5,37 @@ import (
 	"sync"
 
 	"github.com/golang/protobuf/proto"
-	"github.com/zcash-hackworks/lightwalletd/walletrpc"
+	"github.com/zcash/lightwalletd/walletrpc"
 )
 
-type BlockCacheEntry struct {
+type blockCacheEntry struct {
 	data []byte
 	hash []byte
 }
```

Comment:
Identifiers that start with a capital letter are exported, but this type definition can remain private.